### PR TITLE
Add mistral medium latest to model prices

### PR DESCRIPTION
**Pull request by dragosMC91**
I was using mistral latest and noticed that there were no costs calculated after each call

### CODE REVIEW

File: litellm/model_prices_and_context_window_backup.json [hunk 1]
```diff
@@ -603,6 +603,13 @@
         "litellm_provider": "mistral",
         "mode": "chat"
     },
+    "mistral/mistral-medium-latest": {
+        "max_tokens": 8192,
+        "input_cost_per_token": 0.00000273,
```
Comment (ishaan-jaff) [inline litellm/model_prices_and_context_window_backup.json]: should be `27` and `81` based on this https://docs.mistral.ai/platform/pricing/

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
changes requested

File: litellm/model_prices_and_context_window_backup.json (hunk 1 above)
Comment (dragosMC91) [reply litellm/model_prices_and_context_window_backup.json]: I apologise for the mistake, I just did a quick api call to `https://api.mistral.ai/v1/models` and saw that `mistral-medium-latest` / `mistral-medium` look the same and I assumed they cost the same, will correct this after work.
Also, where did you find the pricing for `mistral/mistral-medium` 
```
        "input_cost_per_token": 0.00000273,
        "output_cost_per_token": 0.00000820,
````
I don't see it anywhere in the docs, wondering if that is still correct

File: litellm/model_prices_and_context_window_backup.json [hunk 2]
```diff
@@ -591,15 +591,36 @@
     },
     "mistral/mistral-small": {
         "max_tokens": 8192,
-        "input_cost_per_token": 0.00000066,
-        "output_cost_per_token": 0.00000197,
+        "input_cost_per_token": 0.000002,
+        "output_cost_per_token": 0.000006,
```
Comment (dragosMC91) [inline litellm/model_prices_and_context_window_backup.json:595]: this was indeed the price at one point (jan -> https://web.archive.org/web/20240103230150/https://docs.mistral.ai/platform/pricing/)
but docs now state that the small model pricing has changed -> https://docs.mistral.ai/platform/pricing/

File: litellm/model_prices_and_context_window_backup.json [hunk 3]
```diff
@@ -591,15 +591,36 @@
...
         "litellm_provider": "mistral",
         "mode": "chat"
     },
     "mistral/mistral-medium": {
         "max_tokens": 8192,
-        "input_cost_per_token": 0.00000273,
-        "output_cost_per_token": 0.00000820,
+        "input_cost_per_token": 0.0000027,
+        "output_cost_per_token": 0.0000081,
```
Comment (dragosMC91) [inline litellm/model_prices_and_context_window_backup.json:609]: this difference was coming from the conversion error from euro -> dollar most likely.
initially prices were expressed just in Eur -> https://web.archive.org/web/20240103230150/https://docs.mistral.ai/platform/pricing/
and 2.5 eur ~= 2.72 $

**Comment by ghost** [comment]
LGTM!

**Pull request merged**